krb5wrap.h

```diff
@@ -72,15 +74,16 @@ class KRB5Keytab {
 public:
     KRB5Keytab(const std::string &keytab_name) : m_keytab() {
         krb5_error_code ret = krb5_kt_resolve(g_context, keytab_name.c_str(), &m_keytab);
-        if (ret)
-            throw KRB5Exception("krb5_kt_resolve", ret);
+        if (ret) {
+            krb5_error_exit("krb5_kt_resolve",ret);
+        }
```

Comment:
Why did you choose to replace the exception with a forced exit in the ctor as well? While not used today, being able to catch and handle a failure at this stage looks reasonable to me.